sound/soc/sof/sof-client-probes.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SOF_CLIENT_PROBES_H
+#define __SOF_CLIENT_PROBES_H
+
+struct snd_compr_stream;
+struct snd_compr_tstamp;
+struct snd_compr_params;
+struct sof_client_dev;
+struct snd_soc_dai;
+
+/* Platform callbacks */
+struct sof_probes_host_side_ops {
```

Comment:
do we really need the "side" there? Wouldn't just `sof_probes_host_ops` be enough?

Reply:
I wanted to emphasize that the probes support in some platforms (well, the only one supported at: HDA) is split between DSP and host side control, thus the generic SOF side needs to call to host side ops to get it working.

I can drop the `host_side` for a simple `host` and while doing that I can add a bigger comment block to explain the role.